app/controllers/location_admins/registration_controller.rb

```diff
@@ -0,0 +1,23 @@
+class LocationAdmins::RegistrationController < Devise::RegistrationsController
```

Comment:
This controller is not used at all. It can be used in the future for allowing donors to log in and generate their own receipts